miniforge-cuda/Dockerfile

```
@@ -91,6 +91,7 @@ RUN if [ "${LINUX_VER:0:6}" == "centos" ] ; then \
 # Install miniforge and configure
 RUN wget --quiet ${MINIFORGE_URL} -O /miniforge.sh \
     && /bin/bash /miniforge.sh -b -p /opt/conda \
+    && conda update --all --yes \
```

Would suggest running `conda clean` as well to make sure there are no lingering unused artifacts after the update.

```suggestion
    && conda update --all --yes \
    && conda clean -tipy \
```